cpp/scripts/run-cmake-format.sh

```diff
@@ -45,6 +44,7 @@ fi
 
 DEFAULT_FORMAT_FILE_LOCATIONS=(
   "${KVIKIO_BUILD_DIR:-${HOME}}/_deps/rapids-cmake-src/cmake-format-rapids-cmake.json"
+  "${KVIKIO_BUILD_DIR:-cpp/build}/latest/_deps/rapids-cmake-src/cmake-format-rapids-cmake.json"
```

Comment:
Is this just a consequence of rapids-cmake running in different locations?

Reply:
Yeah, this is to support running `cmake-format` in devcontainers. The devcontainers build in subdirs of `<lib_dir>/build/` based on the CUDA, python, and build type, but there's always a symlink `build/latest` that points to the most recently configured build for a given library.